openapi/shopify.admin/types.bal

```diff
@@ -134,7 +134,7 @@ public type OrderAdjustment record {
     # The unique identifier for the refund that the order adjustment is associated with.
     int? refund_id?;
     # The value of the discrepancy between the calculated refund and the actual refund. If the kind property's value is shipping_refund, then amount returns the value of shipping charges refunded to the customer.
-    int? amount?;
+    string? amount?;
```

Comment:
Have we verified whether we should support both `int` and `string` (union), to be on the safe side? I'm seeing some amounts as integers in https://shopify.dev/docs/api/admin-rest/2023-07/resources/refund#resource-object